src/main/java/org/folio/bulkops/util/UnifiedTableHeaderBuilder.java

```diff
@@ -83,7 +83,7 @@ private static Cell toUnifiedTableCell(Field field, Set<String> forcedVisible) {
     var column = field.getAnnotation(CsvCustomBindByName.class).column();
     return new Cell()
       .dataType(field.getAnnotation(UnifiedTableCell.class).dataType())
-      .value(ADMINISTRATIVE_NOTES.equalsIgnoreCase(column) ? ADMINISTRATIVE_NOTE : column)
+      .value(CSV_COLUMN_NAME_ADMINISTRATIVE_NOTE.equalsIgnoreCase(column) ? TABLE_HEADER_VALUE_ADMINISTRATIVE_NOTE : column)
```

Comment:
it looks a bit strange, if column = Administrative note -> set Administrative note, looks weird

Reply:
removed

Reply:
are you sure it can be removed? what does worker generate?

Reply:
worker doesn't take a part in such a case
![image](https://github.com/folio-org/mod-bulk-operations/assets/141402754/781dc37a-820d-4e1c-8db8-305f01c137a0)

Reply:
worker generates file... check headers there and you'll understand why this function was implemented

Reply:
The snapshot above shows the exact place the table cell value is taken. the source of data is the file, produced by The worker.
The data mapping based on the position and entity type. Then the code analyzes the class field content by exact annotation.

for the HoldingRecord class field with position 18 the CsvCustomBindByName annotation column-attribute value is Administrative note

Reply:
again, worker generates another header, I'm 90% sure you'll introduce new issue? could you check all places in flow?